app/hooks/useWalletCompat.ts

```
@@ -39,15 +40,11 @@ function useWalletCompatInner() {
   const { ready, authenticated, user, logout } = usePrivy();
   const { wallets } = useWallets();
   const { signTransaction: privySignTransaction } = useSignTransaction();
+  const { preferredAddress } = usePreferredWallet();
```

Biome (2.4.4)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)